core/src/net/sf/openrocket/simulation/SimulationOptions.java

```diff
@@ -403,6 +403,10 @@ public void setRandomSeed(int randomSeed) {
 		 */
 		//		fireChangeEvent();
 	}
+
+	public double getMaxLaunchRodAngle(){
+		return this.MAX_LAUNCH_ROD_ANGLE;
```

Comment:
Might be better to have this as a `public static final` field, so when using it we can just use `SimulationOptions.MAX_LAUNCH_ROD_ANGLE`